Refresh.GameServer/Types/Assets/AssetFlags.cs

```diff
@@ -0,0 +1,114 @@
+namespace Refresh.GameServer.Types.Assets;
+
+[Flags]
+public enum AssetFlags
+{
+    None = 0,
+    /// <summary>
+    /// This asset can be dangerous to end users.
+    /// </summary>
+    Dangerous = 1 << 0,
+    /// <summary>
+    /// This asset is a media-type asset, e.g. a PNG or TEX.
+    /// </summary>
+    Media = 1 << 1,
+    /// <summary>
+    /// This asset will only ever be created by mods.
+    /// </summary>
+    Modded = 1 << 2,
+}
+
+public static class AssetSafetyLevelExtensions
+{
+    public static AssetFlags FromAssetType(GameAssetType type, GameSerializationMethod? method)
+    {
+        return type switch
+        {
+            // Non-binary levels will not be loadable 
+            GameAssetType.Level => method == GameSerializationMethod.Binary
+                ? AssetFlags.None
+                : AssetFlags.Dangerous | AssetFlags.Modded,
+            GameAssetType.StreamingLevelChunk => method == GameSerializationMethod.Binary
+                ? AssetFlags.None
+                : AssetFlags.Dangerous | AssetFlags.Modded,
```

Comment:
This is annoying to read... why are some of these values split into multiple while others are not?

I'd also statically import GameSerializationMethod and/or AssetFlags to make this slightly easier to read.

Reply:
> why are some of these values split into multiple while others are not?

I havent fully set the classification for all of them yet, but basically all asset type are "modded if not binary" (aside from textures ofc)

In this case, if we drop the "modded if not binary", then it will just be "dangerous if not binary", since the game is incapable of loading these assets when serialized in text form, and will cause an error for users if uploaded

Reply:
Meant to put the word lines there; this is a readability concern.

Some of these values are split into multiple lines. Most values are not; I don't see a reason for the inconsistency

Reply:
> Meant to put the word lines there; this is a readability concern.
> 
> Some of these values are split into multiple lines. Most values are not; I don't see a reason for the inconsistency

sometimes rider would format it one way, sometimes it wouldn't, i need to run rider's code formatter over this file before merge